frontend/src/store/modules/compartments.js

```diff
@@ -15,13 +15,9 @@ const actions = {
     const compartmentSummary = await compartmentsApi.fetchCompartmentSummary(payload);
     commit('setCompartmentSummary', compartmentSummary);
 
-    commit('maps/setAvailableMaps', {
-      '2d': {
-        compartment: compartmentSummary.compartmentSVGs,
-        subsystem: [],
-      },
-      '3d': { compartment: [{ id: compartmentSummary.info.id, customName: compartmentSummary.info.name }], subsystem: [] },
-    }, { root: true });
+    commit('maps/setAvailableMaps', [
+      ...compartmentSummary.compartmentSVGs,
+    ], { root: true });
```

Comment:
Would this give the same result?

```suggestion
    commit('maps/setAvailableMaps', compartmentSummary.compartmentSVGs, { root: true });
```